src/Test/EntityManagerMockFactory.php

```diff
@@ -20,12 +20,21 @@
 use Doctrine\ORM\EntityRepository;
 use Doctrine\ORM\QueryBuilder;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\EntityManagerMockFactory class is deprecated since 1.x in favor of '.
```

Comment:
```suggestion
    'The '.__NAMESPACE__.'\EntityManagerMockFactory class is deprecated since sonata-project/doctrine-extensions 1.x in favor of '.
```